src/EntityLinking/entity_linkings.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+
+class EntityLinkings:
+
+    def __init__(self):
+        self.imported = time.time()
+        self._entity_mappings = {}
+
+    def add(self, embedding_tag, knowledgebase_id):
```

Comment:
```suggestion
    def add(self, knowledgebase_id, embedding_tag):
```
For less confusion in which way the actual mapping works